int8 cmd
Edge edgeInfo
